pkg/agent/flowexporter/connections/connections.go

```diff
@@ -26,37 +26,37 @@ import (
 	"antrea.io/antrea/pkg/agent/proxy"
 )
 
-type connectionStore struct {
-	connections   map[flowexporter.ConnectionKey]*flowexporter.Connection
+type ConnectionStore struct {
+	Connections   map[flowexporter.ConnectionKey]*flowexporter.Connection
 	ifaceStore    interfacestore.InterfaceStore
 	antreaProxier proxy.Proxier
-	mutex         sync.Mutex
+	Mutex         sync.Mutex
```

Comment:
It is not good to make mutex lock public.

Reply:
Got it. Will try to not using lock out of the package. Need to do some changes on deny_connection side as well.

Reply:
I did not mean to move it out of the package. Lock should be called by methods in struct and others should call those methods.

Reply:
Got your point. Some changes need to be made on deny connections is because currently we are calling `ResetConnStatsWithoutLock` and `DeleteConnWithoutLock` in exporter.go, and we should change those to avoid calling deny connection map lock in exporter.go. I'm working on those changes.